pkg/metrics/metrics.go

```diff
@@ -243,13 +249,20 @@ func NewMultiHistogram(h ...ScalableHistogram) MultiHistogram {
 	return MultiHistogram(h)
 }
 
-// Start implements ScalableHistogram.
+// Start implements ScalableHistogram. Not atomic in time start for each histogram.
 func (h MultiHistogram) Start() {
 	for _, histogram := range h {
 		histogram.Start()
 	}
 }
```

Comment:
I thought the whole point of the last change was to fix that method? why not:

```
now := time.Now()
for _, histogram := range h {
	histogram.StartAt(now)
}
```
?